cpu/msp430fxyz/periph/spi.c

```diff
@@ -52,18 +52,21 @@ int spi_init_master(spi_t dev, spi_conf_t conf, spi_speed_t speed)
     SPI_DEV->TCTL = (USART_TCTL_SSEL_SMCLK | USART_TCTL_STC);
     /* set polarity and phase */
     switch (conf) {
+        case SPI_CONF_FIRST_RISING:
+            SPI_DEV->TCTL |= (USART_TCTL_CKPH & ~(USART_TCTL_CKPL));
```

Comment:
the second part is without effect, simply `SPI_DEV->TCTL |= (USART_TCTL_CKPH);` is enoug?!